galasa-ui/src/app/page.tsx

```diff
@@ -52,21 +55,28 @@ export default function HomePage() {
     return content;
   };
 
-  // Fetches markdown content from local project files, which will be used as the
-  // default content if the service.welcome.markdown property is not set
-  const fetchDefaultMarkdownContent = async () => {
+  const fetchDefaultMarkdownContent = async (locale: string): Promise<MarkdownResponse> => {
+    const localizedFileName = `home-contents.${locale}.md`;
+    const fallbackFileName = `home-contents.md`;
+
+    const markdownDir = path.join(process.cwd(), "public", "static", "markdown");
+
+    // Check if locale-specific file exists
+    const localizedFilePath = path.join(markdownDir, localizedFileName);
+    const fallbackFilePath = path.join(markdownDir, fallbackFileName);
+
+    const fileToRead = fs.existsSync(localizedFilePath)
+      ? localizedFilePath
+      : fallbackFilePath;
```

Comment:
This looks good - can we add a unit test to check that the correct file is selected based on the locale provided?